src/components/orders/OrderPriceDisplay/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import styled from 'styled-components'
+import { faExchangeAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import { formatSmart, safeTokenName, TokenErc20 } from '@gnosis.pm/dex-js'
+
+import { getOrderExecutedPrice, getOrderLimitPrice, RawOrder } from 'api/operator'
```

Comment:
I still think it's strange that these utils are in the API. Being used in the component should be reason enough to move them to the utils. 

The 1st thing I thought was, that the component was getting the prices from the API, so I was concerned. But those are static functions...

Alternatively, if those prices are used everywhere, maybe we should have them as part of the data

Reply:
As I said before, those utils are intended to be what in an OO API are object methods.

> Alternatively, if those prices are used everywhere, maybe we should have them as part of the data

The initial approach was to work with the Raw API order data, but I have nothing against adding those calculated values to a `InternalOrder` type, or whatever we call it.

Reply:
I'll send out a different PR for addressing this part

Reply:
https://github.com/gnosis/gp-ui/pull/161